lib/startup.js

```diff
@@ -0,0 +1,16 @@
+Meteor.startup(function() {
+
+  // Here, we make sure that our collection MaintMode is available and is
```

Comment:
haha. I had this in there earlier. Thanks for fixing my mistake and getting it back in. I also noticed this is an improvement.